src/systemTest/resources/system-test-hub.properties

```diff
@@ -3,3 +3,10 @@ helm.chart.path=http://ddt-nexus.flightstats.io/repository/helm/hub.tgz
 #helm.release.delete=false
 hub.url=http://hub-%s.dls.dev.flightstats.io
 callback.url=http://callback-server-%s.dls.dev.flightstats.io
+
+#### s3
+s3.url=http://s3-%s.dls.dev.flightstats.io
+s3.region=us-east-1
+s3.credentials.path=/etc/.aws/credentials
+s3.bucket.name=hub_%s-local
+s3.mock.enabled=false
```

Comment:
localstack is more advanced than other mocks I have used and the pathStyleAccessEnabled flag will only be useful if running against a localhost s3 mock so I am considering this overkill for now and think I should remove it